core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/CollectionUtil.java

```diff
@@ -46,4 +47,36 @@ private CollectionUtil() {
 
         return treeSet;
     }
+
+    public static boolean containsAny(final Collection collection, final Object[] objects) {
```

Comment:
Is it a good idea to write these ourselves? There are several other solutions from the libraries:

```
Collections.disjoint(collection, objects); // pure Java libraries
CollectionUtils.containsAny(collection, object); // Apache Commons
```